packages/jest-core/src/TestScheduler.ts

```diff
@@ -336,12 +336,12 @@ class TestScheduler {
   async _setupReporters() {
     const {collectCoverage: coverage, notify, verbose} = this._globalConfig;
     const reporters = this._globalConfig.reporters || [['default', {}]];
-    let summary = false;
+    let summaryOptions;
```

Comment:
this should have a type annotation

Reply:
Sure, the type definition is currently in `SummaryReporter.ts` though. Shall I move it to `jest-reporters/src/types.ts` so I can reference it here?

Reply:
Better simply export it from `@jest/reporters` package through `index.ts` file.

Reply:
agreed

Reply:
Let me know what you think of the latest changes.
I'm not used to typescript at all so I'm not sure about it!